main/zoe/guide/contracts/atomic-swap.md

```diff
@@ -57,41 +53,55 @@ const {
 } = await zoe.escrow(aliceOfferRules, alicePayments);
 ```
 
-Alice then makes an offer using the escrowReceipt and tries to collect her payout:
+In order for Alice to escrow with Zoe she needs to redeem her invite. Once Alice redeems her invite she will receive a `seat` and a promise that resolves to her payout.
 
 ```js
-swap.makeFirstOffer(escrowReceipt);
-const payments = await payout;
+const { seat: aliceSeat, payout: alicePayoutP } = await zoe.redeem(
+  aliceInvite,
+  aliceOfferRules,
+  alicePayments,
+);
 ```
 
-She then spreads the `instanceHandle` widely. Bob hears about Alice's
-contract and he decides to look up the `instanceHandle` to see if it
-matches Alice's claims.
+Alice then makes the first offer in the swap:
 
 ```js
+const newInviteP = aliceSeat.makeFirstOffer();
+```
+
+She then spreads the invite widely. Bob hears about Alice's
+contract and he to look up the invite to see if it matches Alice's claims.
+
+```js
+const inviteAssay = zoe.getInviteAssay();
+const bobExclusiveInvite = await inviteAssay.claimAll(newInviteP);
+const bobInviteExtent = bobExclusiveInvite.getBalance().extent;
+
 const {
-  instance: swap,
-  installationHandle,
-  terms,
-} = zoe.getInstance(instanceHandle);
+  installationHandle: bobInstallationId,
+  terms: bobTerms,
+} = zoe.getInstance(bobInviteExtent.instanceHandle);
+
 
 // Bob does checks
-insist(installationHandle === swapInstallationHandle)`wrong installation`;
-insist(sameStructure(terms.assays, swapAssays)`wrong assays`;
+t.equals(bobInstallationId, installationHandle);
```

Comment:
`t.equals` is coming from our test framework, tape, which Bob probably wouldn't have. Let's change these to use `insist` with an expression which should be true.

Reply:
I've never utilized `insist` so I followed the structure above. Let me know if it needs correcting:

```
insist(bobInstallationId === installationHandle)`wrong installation`;
insist(bobTerms.assays === inviteAssay)`wrong assays`;
insist(bobInviteExtent.offerMadeRules === aliceOfferRules.payoutRules)`wrong payoutRules`;
```

Reply:
That looks like correct usage of insist to me. Thanks for making this change! I think the second insist statement comparison needs to either be both arrays or both a single assay.

Reply:
```
insist(bobTerms.assays[0] === inviteAssay)`wrong assays`;
```